src/_06_Intro_To_Hash_Maps/_02_LogSearch.java

```diff
@@ -60,44 +62,28 @@ else if(buttonPressed == searchButton) {
 			}	
```

Comment:
on line 61- just returning the name of a student doesn't really tell the user anything. Try adding on a phrase like: "the student with this ID is:" or "student with ID (ID number) is". 🍰